style.css

```diff
@@ -1 +1,234 @@
+}
+
+header.open {
+  margin-bottom: 16rem;
+}
+header nav ul {
+  display: flex;
+  flex-direction: column;
+  background-color: #ffe5d9;
+}
+header nav li {
+  border-bottom: 1px dotted #000;
+  list-style-type: none;
+  padding: 1rem;
+}
+header nav a {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+header .hamburger {
+  position: absolute;
+  top:20px;
+  right: 20px;
+}
+header .logo {
+  height: 120px;
+}
+header p {
+  align-self:flex-end;
+  font-size: 1rem;
+  font-style: italic;
+  text-align: right;
+  
+}
+/************* Main styles ********** */
+main {
+  max-width: 90vw;
+  margin: auto;
+}
+.section1 {
+  display: grid;
+  grid-template-columns: 1fr;
+
+}
+.section1 img {
+  margin: 40px auto;
+  object-fit: cover;
+  width: 100%;
+}
+.section1 article {
+  grid-row: 1/2;
+}
+.section1 article p {
+  text-align: justify;
+  font-size: 1rem;
+}
+.section1 h1 {
+  font-size: 1.75rem;
+  font-weight: 600;
+  margin: 3.75rem auto 2.5rem;
+}
+
+.section2 ul {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 20px 26px;
+  justify-items: center;
+ 
+}
+.section2 li {
+  list-style-type: none;
+}
+.section2 img {
+  height: 60vw;
+  width: 80vw;
+  transition: opacity .5s ease;
+}
+.section2 img:hover {
+  opacity: 0.5;
+}
+/************* Footer styles********** */
+footer {
+  background-color:#fae1dd;
+  margin-top: 3.75rem;
+  padding: 1rem;
+}
+footer img {
+  width: 18px;
+}
+footer ul {
+  display: flex;
+  flex-direction: column;
+justify-content: center;}
+footer li {
+  list-style-type: none;
+  justify-self: flex-end
+  
+}
+footer a {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+  border-radius: 50%;
+  padding: 0.5rem;
+  margin: 0.5rem;
+  transition: background-color .5s ease;
+}
+footer a:hover {
+  background-color: #f7eeef;
+}
+footer p {
+  font-size: 1rem;
+  text-align: center;
+}
+/************* Responsive styles********** */
+@media screen  and (min-width:712px){
+  .section2 ul {
+     grid-template-columns: 1fr 1fr;
+  }
+  .section2 img {
+    height: 30vw;
+    width: 40vw;
+  }
+}
+
+
+@media screen and (min-width:1056px){
+  html {
+    font-size: 20px;
+  }
+  body {
+    margin: auto;
+  }
+  header nav {
+    display: block;
+    border-radius: unset;
+    position: absolute;
+    transform: translateX(0);
+    transition: none;
+    top:192px;
+    left:5vw;
+    width: 90vw;
+  }
+  header nav ul {
+    background-color: #ffe5d9b5;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+  header nav li {
+    border-bottom: none;
+    padding: 30px 20px;
+  }
+  header nav a {
+    background-color:#fcd5ce;
+    border-radius: 8px;
+    padding:8px 40px;
+    transition: background-color .5s ease;
+  }
+  header nav a:hover {
+    background-color:#fdc4ba; 
+  }
+  header .hamburger {
+    display: none;
+  }
+  .section1 {
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 2rem;
```

Comment:
Good use of grid here, have you considered using grid-template areas? Personally think its the easiest way to implement grid

Reply:
Thanks for your comment... You are absolutely right, otherwise, it can be difficult to manage the network.. I have to improve using 'grid'..